screens/Home/HomeScreen.tsx

```diff
@@ -44,15 +45,15 @@ export const HomeScreen: React.FC<HomeRouteProps> = props => {
       <LinearGradient
         colors={Theme.Colors.gradientBtn}
         style={Theme.Styles.downloadFabIcon}>
-        <Button
-          testID="downloadIcon"
-          icon={plusIcon}
+        <Pressable
```

Comment:
We need to add testId to Fab icon.
As we are wrapping this in Pressable, just check whether we get it in appium for iOS and Android both.
If not, we might need to add `accessible={false}`

Reply:
added the testid for it.